### PR TITLE
Refresh the software tools page

### LINKED ISSUE #5257: Improved "tools" page
Our current [tool](https://www.worldcubeassociation.org/score-tools) page is not very exhaustive nor well organized.
It would be nice to rework it so that it mentions more tool and hopefully with a better classification.

It's pure static and "boring" html, I'll try to make a quick refresh of its content.

### PR DESCRIPTION
**Pull request by viroulep**
Fixes #5257.

cc @Laura-O @DanielEgdal @Goosly @jonatanklosko.

The change is currently [deployed to staging](https://staging.worldcubeassociation.org/score-tools), but obviously the text for each tool is just a draft, I quickly wrote them from what I could find from the various project's pages.

The goal of this page is just to provide a quick overview of what is used currently in WCA competitions, and that can be linked from more in-depth documents such as those the WQAC is currently working on.

I intentionally did not include an "exhaustive" list of features for each tool, as it may changes over time.
Instead I rely on each tool's guide page to provide an accurate description of what it can do.

tools

### CODE REVIEW

**Comment by Laura-O** [comment]
Thanks a lot for taking the time to put this together. I really like it!

File: WcaOnRails/app/views/static_pages/_software_list.html.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          <h3 class="panel-title">
+            <%= hash[:title] %> <%= wca_icon if hash[:wst] %>
+          </h3>
+        </div>
+        <div class="panel-body">
+          <p><%= hash[:description] %></p>
+          <% if hash[:url] %>
+            <p>
+              <%= icon("fas", "link") %>
+              <%= link_to("Website", hash[:url], target: "_Blank") %>
```
Comment (jonatanklosko) [inline WcaOnRails/app/views/static_pages/_software_list.html.erb]: Nit: works either way, but it's the first time I see uppercase `_Blank`. Just pointing out ^^

File: WcaOnRails/app/views/static_pages/score_tools.html.erb [hunk 2]
```diff
@@ -1,13 +1,82 @@
...
+      wst: true,
+    }.merge(t("score_tools.tools.nightmare")),
+  },
+  after: {
+    matcher: {
+      url: "https://viroulep.github.io/scrambles-matcher/",
+      src_url: "https://github.com/viroulep/scrambles-matcher/",
+      maintainer: "Philippe Virouleau",
+      wst: false,
+    }.merge(t("score_tools.tools.matcher")),
```
Comment (jonatanklosko) [inline WcaOnRails/app/views/static_pages/score_tools.html.erb]: This is really neat =)

**Comment by AlexKatyukov** [comment]
Certificates generator is for `During the competition`, maybe `After the competition` but not for `Before the competition`.
Maybe we can add https://bestsiteever.ru/cutifier/? It's convenient sometimes.

File: WcaOnRails/app/views/static_pages/_software_list.html.erb (hunk 1 above)
Comment (viroulep) [reply WcaOnRails/app/views/static_pages/_software_list.html.erb]: Ah no real reason actually, I'm not sure why I wrote it this :p

**Comment by viroulep** [comment]
> Certificates generator is for During the competition, maybe After the competition but not for Before the competition.

We discussed this with Manu actually, I would generally print empty certificates template before the competition and fill them on the spot (I actually printed very rarely the certificates during the competition, maybe like 1 time out of 10 competitions).
Given it was created with the idea of using it during the competition I've indeed moved it.

> Maybe we can add https://bestsiteever.ru/cutifier/? It's convenient sometimes.

By linking to non-WST maintained tools it's fairly expected to have more and more requests like this, and we don't have a clear line to accept or reject linking to tools yet.
However I think all tools linked there should integrate nicely with the WCA website (ie: at least provide a way to log in with our WCA account to get the data the tool needs), and most importantly should be [free software](https://en.wikipedia.org/wiki/Free_software).
It's very unclear to me what's the license of the project, and I couldn't manage to find its sources (I did look into Roman's github, but only saw the previous application he made).

**Pull request merged**